lnrpc/lightning.proto

```diff
@@ -3191,16 +3192,19 @@ message Invoice {
     // Whether this invoice has been fulfilled
     bool settled = 6 [deprecated = true];
 
-    // When this invoice was created
+    // When this invoice was created.
```

Comment:
nit: I think we should turn those into proper multi-line comments (`/* ... */`) now that they span two lines. Not sure if the API docs generator can handle them otherwise.